src/runtime/auto-prompt-manager-test.js

```diff
@@ -443,6 +447,257 @@ describes.realWin('AutoPromptManager', (env) => {
     });
   });
 
+  it('should not set frequency cap local storage if experiment is enabled and content is locked', async () => {
```

Comment:
Can you add one more case to cover when experiment is not enabled?

Reply:
Added test where experiment is explicitly disabled.